harness/determined/common/experimental/user.py

```diff
@@ -67,12 +67,26 @@ def change_display_name(self, display_name: str) -> None:
         resp = bindings.patch_PatchUser(self._session, body=patch_user, userId=self.user_id)
         self._reload(resp.user)
 
+    def change_remote(self, remote: bool) -> None:
+        patch_user = bindings.v1PatchUser(remote=remote)
+        resp = bindings.patch_PatchUser(self._session, body=patch_user, userId=self.user_id)
+        self._reload(resp.user)
+
+    def change_admin(self, admin: bool) -> None:
+        patch_user = bindings.v1PatchUser(admin=admin)
+        resp = bindings.patch_PatchUser(self._session, body=patch_user, userId=self.user_id)
+        self._reload(resp.user)
+
```

Comment:
Were these intended to be left even though they're now unused?

Reply:
Yeah, they aren't needed. Removed.